coverage-tests.js

```diff
@@ -1604,6 +1604,23 @@ test('check window fully with html scrollRootElement after scroll', {
   }
 })
 
+test('check window fully with html scrollRootElement after scroll when fail to scroll', {
+  page: 'Demo',
```

Comment:
I don't think the name `Demo` makes it more clear than it was. Why this page? If it doesn't matter let's use the `Default`, but probably it matters and I would like to get some information about the page from its name.